WebRoot/js/zimbraAdmin/servers/view/ZaEditVolumeXDialog.js

```diff
@@ -41,30 +41,72 @@ ZaEditVolumeXDialog.prototype = new ZaXDialog;
 ZaEditVolumeXDialog.prototype.constructor = ZaEditVolumeXDialog;
 ZaEditVolumeXDialog.helpURL = location.pathname + ZaUtil.HELP_URL + "managing_servers/adding_a_new_storage_volume_to_the_server.htm?locid="+AjxEnv.DEFAULT_LOCALE;
 
-ZaEditVolumeXDialog.prototype.getMyXForm = 
-function() {	
+ZaEditVolumeXDialog.prototype.getMyXForm = function (params) {
 	var xFormObject = {
-		numCols:1,
-		items:[
-			{type:_ZAWIZGROUP_, isTabGroup:true,
-				items:[
-					{ref:ZaServer.A_VolumeName, type:_TEXTFIELD_, label:ZaMsg.LBL_VM_VolumeName, labelLocation:_LEFT_, width:250, visibilityChecks:[],enableDisableChecks:[]},
-					{ref:ZaServer.A_VolumeRootPath, type:_TEXTFIELD_, label:ZaMsg.LBL_VM_VolumeRootPath, labelLocation:_LEFT_, width:250, visibilityChecks:[],enableDisableChecks:[]},
-					{ref:ZaServer.A_VolumeType, type:_OSELECT1_, choices:ZaServer.volumeTypeChoices,width:250, label:ZaMsg.LBL_VM_VolumeType, visibilityChecks:[],enableDisableChecks:[]},
-					{ref:ZaServer.A_VolumeCompressBlobs,
-						type:_WIZ_CHECKBOX_, label:ZaMsg.VM_VolumeCompressBlobs,
-						trueValue:true, falseValue:false, visibilityChecks:[],enableDisableChecks:[]
+		numCols: 1,
+		items: [
+			{
+				type: _ZAWIZGROUP_,
+				isTabGroup: true,
+				items: [
+					{
+						ref: ZaServer.A_VolumeName,
+						type: _TEXTFIELD_,
+						label: ZaMsg.LBL_VM_VolumeName,
+						labelLocation: _LEFT_,
+						width: 250,
+						visibilityChecks: [],
```

Comment:
`visibilityChecks: []` remains here but removed from other items. Which is appropriate?

Reply:
I missed that. It's updated now.